docs/book/src/component-config-tutorial/testdata/project/config/certmanager/certificate.yaml

```diff
@@ -1,25 +0,0 @@
-# The following manifests contain a self-signed issuer CR and a certificate CR.
```

Comment:
Why were these files deleted?

Reply:
good catcher. It is a mistake.

Reply:
Those files were mopve to create webhook in a previous PR IIRC.